app/views/layouts/application.html.erb

```diff
@@ -10,7 +10,17 @@
   </head>
 
   <body>
+
+    <% if notice %>
```

Comment:
if/elsif doesn't work, you would need 4 cases:
notice && alert
!notice && alert
notice && !alert
!notice && !alert